app/models/question.rb

```diff
@@ -7,15 +7,17 @@
 #  created_at                           :datetime         not null
 #  updated_at                           :datetime         not null
 #  question_type                        :string
+#  number                               :integer
 #
 
 class Question < ActiveRecord::Base
-  has_one :question_order
+  has_many :question_orders, inverse_of: :question
   has_one :survey
   has_many :responses
   has_many :response_choices
 
   validates :text, presence: true
+  validates :number, presence: true, numericality: { only_integer: true }
 
-  validates :question_type, presence: true, inclusion: {in: %w[short_answer mc checkbox t_or_f]}
+  validates :question_type, presence: true, inclusion: {in: %w[short_answer multiple_choice true_false]}
```

Comment:
Removed because I didn't see it as an option in the API doc from frontend. Given how easy it is to add in at a later date, I think we should leave it out if we're uncertain for now.